src/NuGet.Services.Validation.Orchestrator/ContentScan/ContentScanValidator.cs

```diff
@@ -0,0 +1,132 @@
+            _contentScanEnqueuer = contentScanEnqueuer ?? throw new ArgumentNullException(nameof(contentScanEnqueuer));
+
+            if (configurationAccessor == null)
+            {
+                throw new ArgumentNullException(nameof(configurationAccessor));
+            }
+            if (configurationAccessor.Value == null)
+            {
+                throw new ArgumentException($"{nameof(configurationAccessor.Value)} property is null", nameof(configurationAccessor));
+            }
+            _configuration = configurationAccessor.Value;
+
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+
+            configurationAccessor = configurationAccessor ?? throw new ArgumentNullException(nameof(configurationAccessor));
+
+            if (configurationAccessor.Value == null)
+            {
+                throw new ArgumentException($"{nameof(configurationAccessor.Value)} property is null", nameof(configurationAccessor));
+            }
+
+            _configuration = configurationAccessor.Value;
+        }
+
+        public async Task<IValidationResponse> StartAsync(IValidationRequest request)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+
+            var validatorStatus = await GetValidatorStatusAsync(request);
+
+            if (validatorStatus.State != ValidationStatus.NotStarted)
+            {
+                _logger.LogWarning(
+                    "Scan and Sign validation with validation Id {ValidationStepId} has already started.",
+                    request.ValidationStepId);
+
+                return GetValidationResponse(validatorStatus);
+            }
+
+            await _contentScanEnqueuer.EnqueueContentScanAsync(request.ValidationStepId, request.InputUrl);
+
+            var result = await _validatorStateService.TryAddValidatorStatusAsync(request, validatorStatus, ValidationStatus.Incomplete);
+
+            return GetValidationResponse(result);
+        }
+
+        public async Task CleanUpAsync(IValidationRequest request)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+        }
+
+        public async Task<IValidationResponse> GetResponseAsync(IValidationRequest request)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+
+            var result = await GetValidatorStatusAsync(request);
+
+            return GetValidationResponse(result);
+        }
+
+        private async Task<ValidatorStatus> GetValidatorStatusAsync(IValidationRequest request)
+        {
+            var validatorStatus = await _validatorStateService.GetStatusAsync(request);
+            return validatorStatus;
```

Comment:
You can just return directly here:

```suggestion
            return await _validatorStateService.GetStatusAsync(request);
```